_constitution/sse_2018_constitution.md

```diff
@@ -5,95 +5,128 @@ title: SSE Constitution
 permalink: about/constitution
 ---
 # Society of Software Engineers Constitution
-_Ratified: July 26, 2016_
+_Ratified: November 2, 2018_
 
 ## DEFINITIONS
+* “Executive board” refers to the President, Vice President, and all Leadership Chairs unless explicitly stated.
+* “Simple majority” refers to greater than 50% of currently assembled members who are able to vote.
+*  “⅔ majority” refers to greater than 66% of currently assembled members who are able to vote.
+
 ## ARTICLE I: Name and Purpose
-### Section 1: Name
-The name of this Organization shall be the Society of Software Engineers, hereafter referred to as the Organization. The official acronym shall be SSE
+###  Section 1: Name
```

Comment:
Minor: This has two spaces after the last '#' which differs from the other headings